xbmc/cores/dvdplayer/DVDPlayer.cpp

```diff
@@ -835,7 +835,7 @@ bool CDVDPlayer::OpenDemuxStream()
   int64_t len = m_pInputStream->GetLength();
   int64_t tim = m_pDemuxer->GetStreamLength();
   if(len > 0 && tim > 0)
-    m_pInputStream->SetReadRate((unsigned int) (g_advancedSettings.m_readBufferFactor * len * 1000 / tim));
```

Comment:
When I see it right, you don't set the readRate at all with the readbufferfactor incorporated.